### PR TITLE
MH3 - 60 cards

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/sage_of_the_unknowable.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Name:Sage of the Unknowable
+ManaCost:2
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/sage_of_the_unknowable.txt:2]: need to confirm cost

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/sage_of_the_unknowable.txt:2]: I know, this one got into the upload by mistake.

File: forge-gui/res/cardsfolder/upcoming/utter_insignificance.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Utter Insignificance
+ManaCost:1 U
+Types:Enchantment Aura
+K:Flash
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Curse
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/utter_insignificance.txt:5]: `Enchant creature` Keyword is missing

File: forge-gui/res/cardsfolder/upcoming/ulamogs_dreadsire.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Ulamog's Dreadsire
+ManaCost:10
+Types:Creature Eldrazi
+PT:10/10
+K:Vigilance
+K:Ward:Sac<1/Permanent.cmcGE1>
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/ulamogs_dreadsire.txt:6]: The Sac Cost probably needs a better description

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/ulamogs_dreadsire.txt]: Wait, I corrected that as a separate commit?

File: forge-gui/res/cardsfolder/upcoming/eldrazi_confluence.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Eldrazi Confluence
+ManaCost:2 C C
+Types:Sorcery
+A:SP$ Charm | Choices$ DBSpatialContortion,DBEldraziDisplacer,DBEldraziScion | CharmNum$ 3 | CanRepeatModes$ True
+SVar:DBSpatialContortion:DB$ Pump | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ +3 | NumDef$ -3 | IsCurse$ True | SpellDescription$ Target creature gets +3/-3 until end of turn.
+SVar:DBEldraziDisplacer:DB$ ChangeZone | TargetMin$ 1 | ValidTgts$ Permanent.nonLand | TgtPrompt$ Select target nonland permanent | RememberTargets$ True | Origin$ Battlefield | Destination$ Exile | SubAbility$ DBReturn | SpellDescription$ Exile target nonland permanent, then return it to the battlefield under its owner's control.
+SVar:DBReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ Exile | Tapped$ True | Destination$ Battlefield
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/eldrazi_confluence.txt:7]: cleanup missing

File: forge-gui/res/cardsfolder/upcoming/guide_of_souls.txt [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+SVar:TrigGainLife:DB$ GainLife | Defined$ You | LifeAmount$ 1 | SubAbility$ DBEnergy
+SVar:DBEnergy:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 1
+T:Mode$ AttackersDeclared | AttackingPlayer$ You | Execute$ TrigImmediateTrig | TriggerZones$ Battlefield | TriggerDescription$ Whenever you attack, you may pay {E}{E}{E}. When you do, put two +1/+1 counters and a flying counter on target attacking creature. It becomes an Angel in addition to its other types.
+SVar:TrigImmediateTrig:AB$ ImmediateTrigger | Cost$ PayEnergy<3> | Execute$ TrigPutCounter | TriggerDescription$ When you do, put two +1/+1 counters and a flying counter on target attacking creature. It becomes an Angel in addition to its other types.
+SVar:TrigPutCounter:DB$ PutCounter | ValidTgts$ Creature.attacking | TgtPrompt$ Select target attacking creature | CounterType$ P1P1 | CounterNum$ 2 | SubAbility$ FlyingCounter
```
[the remark this answers is not in the document]
Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/guide_of_souls.txt:10]: I've noticed that there is a technology to put several different counters on permanent at once (I used it on Gift of the Viper), but it only works if the numbers are the same.

[the remark this answers is not in the document]
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/guide_of_souls.txt:10]: check _Filigree Vector_ for how to script this instead

[the remark this answers is not in the document]
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/guide_of_souls.txt]: still using the wrong effect among other things, I can fix it when we get that other spoiler complete

[the remark this answers is not in the document]
Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/guide_of_souls.txt]: OK, thank you.

**Pull request closed without merging**